docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: YugabyteDB Anywhere node software requirements
+headerTitle: Software requirements for universe nodes
+linkTitle: Software requirements
+description: Prerequisites for installing YugabyteDB.
+headContent: Prepare a VM for deployment in a universe
+menu:
+  preview_yugabyte-platform:
+    identifier: server-nodes-software
+    parent: server-nodes
+    weight: 20
+aliases:
+  - /preview/yugabyte-platform/configure-yugabyte-platform/supported-os-and-arch/
+  - /preview/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes/
+type: indexpage
+---
+
+The Linux OS and other software components on each database cluster node must meet the following minimum software requirements.
+
+Depending on the cloud provider type and permissions you grant, you may have to install all of these requirements manually, or YugabyteDB Anywhere (YBA) will install it all automatically.
+
+##### Linux OS
+
+YBA supports deploying YugabyteDB on a variety of [operating systems](../../../reference/configuration/operating-systems/).
+
+AlmaLinux OS 8 disk images are used by default, but you can specify a custom disk image and OS.
+
+Linux must be configured with a `yugabyte` user, sufficient file descriptor limits (that is, ulimits), and more. With the exception of a fully manual on-premises provisioning, this configuration is performed automatically by YBA.
+
+##### Additional software
+
+YBA requires the following additional software to be pre-installed on nodes:
+
+- Python 3.6-3.8
+- OpenSSH Server. Allowing SSH is recommended but optional. It can be skipped in some on-premises deployment approaches; all other workflows require it.
+- Tar
+- unzip
+- policycoreutils-python
```

Comment:
Minor: lower case would match the typical package names better for the last 3 lines

For OpenSSH Server, capital letters is ok because we are not specifying a package name.